components/panelbar/overview.md

```diff
@@ -121,22 +111,9 @@ To use a Telerik PanelBar for Blazor:
 }
 ````
 
->caption The result from the snippet above
+## Data Binding
 
-![Basic example of panelbar](images/panelbar-basic-example.png)
-
->caption Component namespace and reference
-
-````CSHTML
-@* Get a reference to the PanelBar *@
-
-<TelerikPanelBar @ref="@PanelBarReference">
-</TelerikPanelBar>
-
-@code {
-    private Telerik.Blazor.Components.TelerikPanelBar PanelBarReference { get; set; }
-}
-````
+To show any items, the Blazor PanelBar requires a data source that you can provide through the `Data` property. The PanelBar allows you to display the items both as flat data and hierarchically. [Read more about the Blazor PanelBar data binding...]({%slug panelbar-data-binding-overview%})
 
 ## Elements of a PanelBar Item
```

Comment:
This section can be misleading. When not using templates, "Header" and "Content" are two separate items. We should explain this in some other way. We can also rename the section to "Templates"